docs/sources/mimir/operators-guide/monitor-grafana-mimir/dashboards/ruler/index.md

```diff
@@ -18,7 +18,7 @@ The following example shows a Ruler dashboard from a demo cluster.
 
 ![Grafana Mimir ruler dashboard](mimir-ruler.png)
 
-Note: Even while operating in [Remote ruler mode]({{< relref "../../../architecture/components/ruler/#remote" >}}) you will still see values for the `Read from ingesters - QPS`.
+> **Note:** Even while operating in [Remote ruler mode]({{< relref "../../../../references/architecture/components/ruler/#remote" >}}) you will still see values for the `Read from ingesters - QPS`.
```

Comment:
nit: This should probably use the admonition shortcode now.

We can always change them in a future PR.